root/src/main/java/com/example/servingwebcontent/ConnectDatabase.java

```diff
@@ -15,42 +15,46 @@ public ConnectDatabase (String database, String user, String pass) {
 		}
 	}
 
-	public ResultSet getMovie (String id) {
+	public Movie getMovie (String id) {
 		ResultSet rs = null;
-		String query = "select * from movies where id=?";
+		String query = "select * from movies where " + Movie.getColumns()[0] + " = ?";
+		Movie movie = null;
 		try {
 			PreparedStatement stmt = con.prepareStatement(query);
 			stmt.setString(1, id);
 			rs = stmt.executeQuery();
+			if (rs.next()) {
+				movie = new Movie(rs.getString(Movie.getColumns()[2]), rs.getString(Movie.getColumns()[1]), rs.getString(Movie.getColumns()[2]), rs.getString(Movie.getColumns()[3]), rs.getInt(Movie.getColumns()[4]), rs.getString(Movie.getColumns()[5]));
+			}
 		} catch (SQLException e) { 
-			throw new RuntimeException("failed to query db", e); 
+			throw new RuntimeException("failed to query movies table", e); 
 		}
 
-		return rs;
+		return movie;
 	}
 
-	public ResultSet updateMovie (Movie movie) {
+	public void updateMovie (Movie movie) {
 		ResultSet rs = null;
-		String query = "update movies set type = ?, name = ?, releasedate = ?, duration = ?, filmrating = ? where id = ?";
+		String query = "update moves set " + Movie.getColumns()[1] + " = ?, " + Movie.getColumns()[2] + " = ?, " + Movie.getColumns()[3] + " = ?, " + Movie.getColumns()[4] + " = ?, " + Movie.getColumns()[5] + " = ? where " + Movie.getColumns()[0] + " = ?";
 		int i = 1;
 		try {
 			PreparedStatement preparedStmt = con.prepareStatement(query);
 			preparedStmt.setString (i++, movie.getType());
 			preparedStmt.setString (i++, movie.getName());
 			preparedStmt.setString (i++, movie.getReleaseDate());
-			preparedStmt.setString (i++, movie.getDuration());
+			preparedStmt.setInt (i++, movie.getDuration());
 			preparedStmt.setString (i++, movie.getFilmRating());
-			preparedStmt.setString (i++, movie.getId());
+			preparedStmt.setString (i++, movie.getID());
 			preparedStmt.execute();
 		} catch (SQLException e) {
 			throw new RuntimeException("failed to update movie entry", e);
 		}
 
-		return rs;
+		return;
```

Comment:
if it's a void method, you don't need a return at the end. You would only use a return if you want to exit the method early